inst/tinytest/test_misc.R

```diff
@@ -176,9 +176,14 @@ expect_equal(testNullableString("blah"), "blah")
 expect_true(nchar(Rcpp:::bib()) > 0, info="bib file")
 
 #    test.getRcppVersion <- function() {
-expect_true(inherits(getRcppVersion(), "package_version"), info="package_version object")
-expect_true(getRcppVersion(devel=TRUE) >= getRcppVersion(devel=FALSE), info="dev greater equal release")
+expect_true(inherits(Rcpp::getRcppVersion(), "package_version"), info="package_version object")
+expect_true(Rcpp::getRcppVersion(devel=TRUE) >= Rcpp::getRcppVersion(devel=FALSE), info="dev greater equal release")
 
 ## if need be it can be useful to fail to test e.g. the Docker setup
 ## commented out now as we prefer to pass when not debugging ;-)
 # expect_true(FALSE, info="oh noes")
+
+## test that a message is output as is, and a suppressedMessage is not
+expect_equal(capture.output(messageWrapper("ABCdef"), type="message"), "ABCdef")
```

Comment:
FWIW I think it would be a bit more idiomatic to capture test against the condition object directly with something like:

```
msg <- tryCatch(message("test"), message = identity)
expect_equal(msg$message, "ABCdef\n")
```

Reply:
Interesting. I don't think I have ever seen `tryCatch(... , message = identity)` before so in that sense the suggested code it harder to read for me :)

Reply:
I added this now, and made the (common) payload a variable at the same time:

```r
## test that a message is output as is, and a suppressedMessage is not
txt <- "ABCdef"
expect_equal(capture.output(messageWrapper(txt), type="message"), txt)
expect_equal(capture.output(suppressMessages(messageWrapper(txt)), type="message"), character())
expect_message(messageWrapper(txt))
## test for message component
msg <- tryCatch(message(txt), message = identity)
expect_equal(msg$message, paste(txt, "\n", sep=""))
```